docs/using-profiles.md

```diff
@@ -0,0 +1,43 @@
+(using-lint-profiles)=
+
+# Applying lint profiles
+
+Ansible-lint profiles allow content creators to progressively improve the quality of Ansible playbooks, roles, and collections.
+
+During early development cycles, you need Ansible-lint rules to be less strict.
+Starting with the minimal profile ensures that Ansible can load your content.
+As you move to the next stage of developing content, you can gradually apply profiles to avoid common pitfalls and brittle complexity.
+Then, when you are ready to publish or share your content, you can use the ``shared`` and ``production`` profiles with much stricter rules that harden security and guarantee reliability.
+
+```{note}
+Tags such as `opt-in` and `experimental` do not take effect for rules that are included in profiles, directly or indirectly.
+If a rule is in a profile, Ansible-lint applies that rule to the content.
+```
+
+After you install and configure `ansible-lint`, you can apply profiles as follows:
```

Comment:
@priyamsahoo and @ganeshrn Is it possible to use profiles on the command line only? I tried to set a ``--profile=foo`` in the vscode extension but it didn't work.

Also I'm curious if it's possible to set a profile in the `.ansible-lint` configuration file. I could imagine a usage scenario where a profile might be imposed.